src/group_theory/quotient_group.lean

```diff
@@ -132,6 +132,18 @@ lemma coe_inv (a : G) : ((a⁻¹ : G) : Q) = a⁻¹ := rfl
 @[simp] lemma coe_gpow (a : G) (n : ℤ) : ((a ^ n : G) : Q) = a ^ n :=
 (mk' N).map_gpow a n
 
+omit nN
+
+@[simp] lemma coe_gsmul {G' : Type*} [add_group G'] (N : add_subgroup G') [N.normal] (a : G') (n : ℤ) :
+  ((n • a : G') : quotient_add_group.quotient N) = n • a :=
+(quotient_add_group.mk' N).map_gsmul a n
+
+@[simp] lemma coe_nsmul {G' : Type*} [add_group G'] (N : add_subgroup G') [N.normal] (a : G') (n : ℕ) :
+  ((n • a : G') : quotient_add_group.quotient N) = n • a :=
+(quotient_add_group.mk' N).map_nsmul a n
```

Comment:
I think `to_additive` might be able to generate these if you attach that attribute to `coe_gpow` above (after merging master and solving the merge conflict).

Reply:
Hi Eric, doing to_additive works with coe_pow but there is an error with coe_gpow saying "unknown declaration 'add_monoid_hom.map_gpow'" ```(mk' N).map_gpow a n )is used in the proof```

Reply:
Hi Eric, adding to_additive works for coe_pow but fails for coe_gpow, generating the error: "unknown declaration 'add_monoid_hom.map_gpow'", ``` mk' N.map_gpow a n``` is used in the proof

Reply:
As a follow-up, I added the `to_additive` attribute to coe_pow to auto-generate coe_nsmul. I could not do the same thing for coe_gpow because to_additive was not used in the lemma `add_monoid_hom.map_gpow`.